demumble.cc

```diff
@@ -115,6 +118,7 @@ int main(int argc, char* argv[]) {
           printf("%.*s", static_cast<int>(special), cur);
         else if (need_separator)
           printf("\n");
+
```

Comment:
Can you just not touch the code down here at all please? :-)

Reply:
Heh, sorry. I'll send a fresh PR.